src/models/GatewayAccount.class.js

```diff
@@ -15,6 +15,7 @@ const pendingCredentialStates = [CREDENTIAL_STATE.CREATED, CREDENTIAL_STATE.ENTE
  * @property {boolean} disableToggle3ds
  * @property {string} externalId The external id of the gateway account
  * @property {string} emailCollectionMode The email collection mode of the gateway account
+ * @property {JSON} emailNotifications The email collection notification settings of the gateway account
```

Comment:
is `JSON` a valid type annotation? it's really just an object

Reply:
Is the alternative to make a class for it?